gradle/versions.gradle

```diff
@@ -14,6 +14,7 @@ ext.libVersions =
     commons_lang3: '3.1',
     commons_io: '2.1',
     commons_dbutils: '1.3',
+    dagger: '2.4',
```

Comment:
@tpetracca mentioned this was required. Doesn't seem crazy to me to bump the version since we know the old 2.0.2 version causes problems with Atlas consumers. 

Current version of `com.google.dagger:dagger-compiler` is `2.6.1` released 6 days ago, and `2.4` released 2016-04-21. See https://github.com/google/dagger/releases and 
https://github.com/google/dagger/blob/5a5f70eea46eaca8839530f7b2cff932d1a6b384/CHANGELOG.md for change log since 2.0.2